crdt/atom/atom.go

```diff
@@ -0,0 +1,108 @@
+package atom
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/crdteam/causal-tree/crdt/utils/indexmap"
+)
+
+// AtomValue is a tree operation.
+type AtomValue interface {
```

Comment:
With the package structure, this can be called `Value`, to be referenced as `atom.Value`

Reply:
done.